exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -69,6 +69,11 @@ impl JavaRuntimeProxy {
         }
     }
 
+    /// Provides the same `Executor` with runtime
+    pub fn get_executor(&self) -> &Executor {
```

Comment:
Shall the Runtime have this method? It is currently used by the test code only,
which _itself_ creates the Executor and can use it.